### PR TITLE
[4.0] Convert plugin system redirect to prepared statements

### PR DESCRIPTION
**Pull request by HLeithner**
# Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.

### Testing Instructions
Use the plugin in all ways you can think of.

User com_redirect and tries auto redirect.

### Expected result
Nothing changed.

### CODE REVIEW

File: plugins/system/redirect/redirect.php [hunk 1]
```diff
@@ -142,32 +143,23 @@ public function handleError(ErrorEvent $event)
...
+					$urlRootRel,
+					$urlRootRelSlash,
+					$urlWithoutQuery,
+					$urlRelWithoutQuery,
+					$orgurl,
+					$orgurlRel,
+					$orgurlRootRel,
+					$orgurlRootRelSlash,
+					$orgurlWithoutQuery,
+					$orgurlRelWithoutQuery
```
Comment (SharkyKZ) [inline plugins/system/redirect/redirect.php]:
```suggestion
					$orgurlRelWithoutQuery,
```

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 9d2f9991ddd43c2b793bdf4ef348e3ce3eb8e5f5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25538">issues.joomla.org/tracker/joomla-cms/25538</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9d2f9991ddd43c2b793bdf4ef348e3ce3eb8e5f5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25538">issues.joomla.org/tracker/joomla-cms/25538</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25538">issues.joomla.org/tracker/joomla-cms/25538</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**